config/stackdriver.php

```diff
@@ -88,6 +88,8 @@
                 'workerNum' => 1,
             ],
         ],
+
+        'blacklist' => [],
```

Comment:
It might be better to replace this with `ignored_paths`. For instance, Telescope also uses that key for this goal.

Reply:
Done :)